docs/developer-docs/daos/nns/proposal-lifecycle.mdx

```diff
@@ -0,0 +1,89 @@
+The governance canister stores a neuron's choice in the still empty ballot where the neuron's voting power is already recorded.
+In this way, the proposal always contains the information of the total voting power of "Yes" and "No" votes.
+
+A proposal can be _decided_ in two ways:
+
+1. **Absolute Majority before the voting period ends**: At any point, even before the voting period ends, if an absolute
+majority (more than half of the total voting power stored in the proposal) has voted Yes, then the proposal is adopted and if
+ an absolute majority has voted No, then the proposal is rejected.
+2. **Simple Majority at the voting period’s end**: When the voting period ends, if a simple majority (more than half of the
+cast votes) has voted Yes and the number of these Yes-votes constitute at least 3% of the total voting power, then the
+proposal is adopted. Otherwise, the proposal is rejected.
+
+What also plays into this is an algorithm called _wait-for-quiet_ that dynamically adjusts the deadline.
+The idea of wait-for-quiet is to decide proposals quickly when all voters agree, but increase the time that neurons
+can vote for proposals that are controversial. That means that the voting period can be dynamically increased, depending
+on the neurons’ votes. In particular, each time a proposal’s outcome is turned (either a Yes-majority is turned to a
+No-majority or vice versa), the proposal’s deadline is increased. Currently, a proposal's initial voting period is 4 days
+and can be increased by at most another 4 days. That is, the voting period that is taken into account for the above rules
+can be between 4 and 8 days, depending on the voting activity.
+
+## Proposal execution
+Technically, a proposal defines a method on a canister that will be called if the proposal is adopted with some parameters
+(see [here](./proposal-advanced.mdx) for details).
+When a proposal is adopted, the defined method on the specified canister is automatically called with the given parameters.
+This is done fully automatically by the governance canister and does not require any manual intervention.
+A proposal's execution may be successful, in which case the proposal is labeled as `executed` or it may fail, in which case
+the proposal is labeled as `failed`.
+When a proposal is rejected, most of the time nothing happens (in very few cases some cleanup work is initiated).
+Therefore, `rejected` is also a possible final proposal state.
+
+:::info
+A proposal is adopted or rejected as soon as a decision can be made, which might be before the voting deadline.
+To ensure that neurons can still take their time to vote and get rewards, neurons that vote after the proposals' decision but
+before the voting period's deadline still get rewards.
+See [this page](./staking-voting-rewards.mdx) for more information on rewards.
+
+:::
```

Comment:
```suggestion
Technically, a proposal defines a method on a canister that will be called if the proposal is adopted with [some parameters](./proposal-advanced.mdx).

When a proposal is adopted, the defined method on the specified canister is automatically called with the given parameters. This is done automatically by the governance canister and does not require any manual intervention.
A proposal's execution may be successful, in which case the proposal is labeled as `executed`, or it may fail, in which case the proposal is labeled as `failed`.

When a proposal is rejected, most of the time nothing happens. In very few cases some cleanup work is initiated.
Therefore, `rejected` is also a possible final proposal state.

:::info
A proposal is adopted or rejected as soon as a decision can be made, which might be before the voting deadline.
To ensure that neurons can still take their time to vote and get rewards, neurons that vote after the proposals' decision but before the voting period's deadline still get rewards.
See [staking and voting rewards](./staking-voting-rewards.mdx) for more information on rewards.
:::
```